lib/Models/Terria.ts

```diff
@@ -416,7 +421,12 @@ export default class Terria {
     persistViewerMode: true,
     openAddData: false,
     feedbackPreamble: "feedback.feedbackPreamble",
-    feedbackMinLength: 0
+    feedbackMinLength: 0,
+    extraCreditLinks: [
```

Comment:
would this leave them out of the translation.json flow? It might be better to have the extraCreditLinks be an object so that use the object key to refer to the url and have that same key be referenced in translation.json

Reply:
Good point about translating the text. I have changed the default link text to i18n keys. However I am keeping it as an array as it makes the ordering more explicit (although JS object key ordering is in the lexical order).